specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,68 @@
+CreationPriority
+===
+
+# Background
+WebView2 has a functionality enabled by default which raises the launch priority of WV2 processes to ensure that WV2 processes are given higher CPU priority and access to resources while creation, and that can bring improvements on time spent creating a WV2 instance. Since this improvement is at the cost of taking higher CPU priority, we introduce `CreationPriority` property to give developers an option to decide by themselves.
+
+# Description
+* Setting `CreationPriority` to `CreationPriority::NORMAL` will signal WebView2 to use less system resources during the creation. 
+* Setting `CreationPriority` to `CreationPriority::HIGH` will attempt to enable the launch time improvement functionality with raising the launch priority during creation.  
+* `CreationPriority::DEFAULT` is the default value, we treat it as same as `CreationPriority::HIGH`.
+## Notes
+* Your application needs to verify that the version of Edge is >=95 before applying the `CreationPriority`, it can cause issues if used on earlier versions of Edge.
+* A prerequisitive for WebView2 applying the launch time improvement is the host app's priority is at least normal. Else, WebView2 still get created with normal priority even with `CreationPriority::HIGH`.
+* Currently launch time improvement functionality is only enabled for WebView2 manager process, so the `CreationPriority` only effect the creation of manager process. `CreationPriority` can play the same role for other process types like render process if WebView2 opts in launch time improvement functionality for those types in the future.
+# Examples
+## Win32 C++
+```cpp
+auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+options->put_CreationPriority(CreationPriority::NORMAL);

```

Comment:
I can add a description for use case.
>  Is there sample code we can include here that can make it clear what the end to end scenario is rather than just showing setting the property and calling Create?

`CreationPriority` is only used in creation WebView environment, I don't think there're more reference codes here.